src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java

```diff
@@ -87,6 +87,13 @@
    */
   protected float bicycleSafetyFactor;
 
+  /**
+   * walkSafetyFactor = length * walkSafetyFactor. For example, a 100m street with a safety
+   * factor of 2.0 will be considered in term of safety cost as the same as a 150m street with a
```

Comment:
```suggestion
   * factor of 2.0 will be considered in term of safety cost as the same as a 200m street with a
```
or have I misunderstood this?

Reply:
Yes it seems to be wrong. I also edited the comment for the bicycle safety (from which I had originally copied this from). Although, for the bicycle safety it's a bit more complicated with the elevation changes and optimization types...